components/event-bus/docs/api.yaml

```diff
@@ -119,6 +119,14 @@ components:
           description: ID of the published event
           pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
           example: "31109198-4d69-4ae0-972d-76117f3748c8"
+        status:
+          type: string
+          description: Status os message publishng
+          example: "SENT"
+        reason:
+          type: string
+          description: reason behind the message publishing status
+          example: "Message was ignored since there is no subscriptions to this event"
```

Comment:
We can make it a bit generic. e.g. `Event was ignored as there are no subscriptions or consumers configured for this event`